### PR TITLE
Boycotting, Readme and Installer fix

### PR DESCRIPTION
**Pull request by hendrikmaus**

### CODE REVIEW

File: config/deploy.php [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 date_default_timezone_set('TIMEZONE');
 
-include_once 'vendor/ekandreas/docker-bedrock/recipe.php';
+require_once 'recipe/common.php';
+require_once 'vendor/ekandreas/docker-bedrock/recipe.php';
 
-set('mysql.local', 'the_project');
```
Comment (hendrikmaus) [inline config/deploy.php]: these did not seem to do anything

File: parameters.php [hunk 2]
```diff
@@ -1,7 +0,0 @@
-<?php
```
Comment (hendrikmaus) [inline parameters.php]: the whole file wasn't used in the project

File: src/DockerBedrock/Installer.php [hunk 3]
```diff
@@ -12,16 +12,17 @@ public static function postAutoloadDump(Event $event)
         $projectDir = realpath($vendorDir.'/../');
         $deployContent = file_get_contents(realpath(__DIR__.'/../../config/deploy.php'));
         $deployContent = str_replace('the_project.dev', '', $deployContent);
+        $deployerFile = 'deploy.php';
 
-        if (!file_exists($projectDir.'/deploy.dep')) {
```
Comment (hendrikmaus) [inline src/DockerBedrock/Installer.php]: this kept overwriting the user's deploy.php file on every composer install/update once one adds the script to the composer.json

Comment (ekandreas) [inline src/DockerBedrock/Installer.php]: The meaning with that script is to initialze a deploy file at startup. If at every composer update then of cource remove it!

Comment (hendrikmaus) [inline src/DockerBedrock/Installer.php]: It is fine now, the only issue was the mismatch in the file extension

**Pull request merged**